sdformat_to_mjcf/tests/resources/double_pendulum/double_pendulum-1.5.sdf

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0'?>
+<sdf version="1.5">
+  <model name="double_pendulum_kinematics">
```

Comment:
This hasn't been merged yet. I think we should limit this PR to files that have been reviewed and merged.